### ISSUE TITLE
AIR SDK 5.0.1.3 returns 5.0.1.2 version number

### ISSUE THREAD

**Issue by sjabberwocky** [airsdk/Adobe-Runtime-Support#2304]
AIR SDK 5.0.1.3 returns 5.0.1.2 version number.

**Comment by ajwfrost** [comment]
Well, kind of. The issue is that we've moved to a system where each release is made up of components; the version you see there is from ADT which is in one of the components that hadn't changed between 50.0.1.2 and 50.0.1.3.

The plan to get around this is that we'll ensure that the XML file is correct (from our 'big' zip files, this is easy, and the SDK Manager will be updated to manually sort this out), and then ADT will pick the version up from there.

But yes, good point, and we'll leave this open until we've got the appropriate fix/workaround in place... I might just see what happens to Animate if the version number changes without you changing anything there..!

thanks
